src/content/protocols/catinabox.md

```diff
@@ -0,0 +1,122 @@
+
+- Half of the fee from the v1.1 protocol are used to incentivize BoxETH liquidity, however this operation is fully done by hand by the team and can't be considered autonomous, this operation doesn't have any fail safe and in case of problem those incentives fund would stay stuck forever leading to degraded liquidity. Such a lack of liquidity make the protocol less attractive for new users and increase the risk of redemption for already existing users.
+
+- Fee received in the form of stETH need to be converted to BoxETH to be distributed to BoxFEE staker, to do this the `trade` function need to be called, as of this writting only the team multisig and deployer can do this operation, thus this operation cannot be considered autonomous and should problems arise could result in a chunk of the reward to staker being stuck forever. 
+
+> Autonomy score: M
+
+## Exit Window
+
+Currently the protocol have a "Medium" risk score for upgradeability and no exit window or timelock on any permission.
+
+> Exit Window score: M
+
+## Accessibility
+
+The only front end for the protocol is available at `https://catinabox.finance/app`, no self-hostable front end is provided on github nor is any tutorial on using the protocol through etherscan in the doc.
+
+> Accessibility score: H
+
+# Technical Analysis
+
+## Contracts
+
+| Contract Name          | Address                                    |
+| --------------------   | ------------------------------------------ |
+| CatInABox v1           | 0x7f0A0C7149a46Bf943cCd412da687144b49C6014 |
+| CatInABox v1.1         | 0x6ffD098E92B606b2947b89A08911C00ca06890FA |
+| BoxETH                 | 0x7690202e2C2297bcD03664e31116d1dFfE7e3B73 |
+| Peg Stability Module   | 0x24146D1B3339Cf76b455dC42e71Ea5Cdff4aE0d7 |
+| BoxFEE                 | 0xE4B91fAf8810F8895772E7cA065D4CB889120f94 |
+| BoxFEE staking contract| 0x320c871b6f7721083604FfdD8070E64c1d3c5d7C |
+
+## Permission owners
+
+| Name                                | Account                                       | Type         |
+| ----------------------------------- | --------------------------------------------- | ------------ |
+| Undeclared Multisig (Team Treasury) | [0x98e6475C01D018Ae78c02ef48738f687538226Af](https://etherscan.io/address/0x98e6475C01D018Ae78c02ef48738f687538226Af) | Multisig 3/5 |
+| Undeclared EOA (Team Deployer)      | [0x56C40ade4C96800b13E20A71Ff60c4005F0bCF8e](https://etherscan.io/address/0x56C40ade4C96800b13E20A71Ff60c4005F0bCF8e) | EOA          |
+
+## Permissions
+
+| Contract       | Function           | Impact                                                                             | Owner                     |
+| -------------- | ------------------ | ---------------------------------------------------------------------------------- | ------------------------- |
+| CIAB v1        | setFee             | Change the protocol fee between 25 and 100% of the generated yield                   | Undeclared Multisig       |
```

Comment:
be here more specific on impact, it's the fee switch, thus only between 0 and 75% (the rest is dynamic)